mqtt/mqtt-bridge/src/bridge.rs

```diff
@@ -110,9 +108,9 @@ impl Bridge {
         &self,
         mut topics: HashMap<String, Topic>,
         address: &str,
-        keep_alive: Duration,
-        clean_session: bool,
+        port: Option<String>,
```

Comment:
initially it was like that, but tls connection failed because it is using the address:port to connect to tls stream and cert failed, it needs only the address part when it opens the tls stream. Other option is no parse the address:port and extract the address